lib/target/kc/classic/kc_crt0.asm

```diff
@@ -6,14 +6,15 @@
 ;
 ;		OPTIMIZATIONS:
 ;		#pragma output nosound   - Save few bytes not preserving IX on tape and sound interrupts
+;		#pragma output nointerrupsaving   - do not save some interrupt vectors on startup (includes nosound)
```

Comment:
is `output`still right?

Reply:
https://github.com/z88dk/z88dk/wiki/CRT#pragma-overrides

might test in the assembly. Like...
https://github.com/z88dk/z88dk/wiki/Tool---z80asm---directives#ifdef-name--elifdef-name--else--endif

`IFDEF nointerrupsaving`

or 

https://github.com/z88dk/z88dk/wiki/Tool---z80asm---directives#ifndef-name--elifndef-name--else--endif
`IFNDEF nointerrupsaving`